st2actions/packaging/debian/install

```diff
@@ -1,4 +1,5 @@
 st2actions usr/lib/python2.7/dist-packages/
 st2actions usr/lib/python2.7/dist-packages/
 conf/* etc/st2actions/
+bin usr/lib/python2.7/dist-packages/st2actions/
```

Comment:
I screwed up, it is. Forgot to switch branches back to master. I'll rebase my fork's branch and remove it.